src/components/Modal/Contributors/Contributors.tsx

```diff
@@ -1,11 +1,204 @@
+
+interface Person {
+	name: string;
+	image: string;
+	function?: string;
+	institution: string;
+	hasImage?: boolean;
+}
+
+const mainContributors: Person[] = [
+	{
+		name: 'Vinicius Brei',
+		image: viniciusBrei,
+		function: 'Lead Researcher',
+		institution: 'UFRGS',
+	},
+	{
+		name: 'Alex Sandy Pentland',
+		image: alexSandy,
+		function: 'Principal Investigator',
+		institution: 'MIT',
+	},
+	{
+		name: 'Mohsen Bahrami',
+		image: mohsenBahrami,
+		function: 'Lead Researcher',
+		institution: 'MIT',
+	},
+	{
+		name: 'Alina Flores',
+		image: alinaFlores,
+		function: 'Lead Researcher',
+		institution: 'UFRGS',
+	},
+	{
+		name: 'Ricardo Limongi',
+		image: ricardoLimongi,
+		function: 'Lead Researcher',
+		institution: 'UFG',
+	},
+	{
+		name: 'Leonardo Gomes',
+		image: leonardoGomes,
+		function: 'Lead Software Developer',
+		institution: 'Universidade de Brasília',
+	},
+	{
+		name: 'Justin Anderson',
+		image: justinAnderson,
+		function: 'Lead Software Developer',
+		institution: 'MIT',
+	},
+];
+
+const otherContributors: Person[] = [
+	{
+		name: 'Enzo Gabriel',
+		image: enzoGabriel,
+		institution: 'Universidade de Brasília',
+		hasImage: true,
+	},
+	{
+		name: 'Valderson Junior',
+		image: valdersonJunior,
+		institution: 'Universidade de Brasília',
+		hasImage: true,
+	},
+	{
+		name: 'Ana Luisa',
+		image: anaLuisa,
+		institution: '',
+		hasImage: false,
+	},
+
+	{
+		name: 'Andreon Magagna',
+		image: andreonMagagna,
+		institution: 'UFRGS',
+		hasImage: true,
+	},
+
+	{
+		name: 'Carla Netto',
+		image: carlaNetto,
+		institution: '',
+		hasImage: true,
+	},
+	{
+		name: 'Jaiany Rocha',
+		image: jaianyRocha,
+		institution: '',
+		hasImage: true,
+	},
+	{
+		name: 'Jheniffer Dany Lucas',
+		image: jhenifferLucas,
+		institution: '',
+		hasImage: true,
+	},
+	{
+		name: 'Luís Eduardo Meneguetti',
+		image: luisMenegetti,
+		institution: '',
+		hasImage: true,
+	},
+	{
+		name: 'Thales Martins',
+		image: thalesMartins,
+		institution: '',
+		hasImage: false,
+	},
+	{
+		name: 'Vilmar Oro Boff',
+		image: vilmarBoff,
+		institution: '',
+		hasImage: true,
+	},
+];
 
 const Contributors = () => {
-    return (
-      <ModalContainer title="Contributors">
-        <></>
-      </ModalContainer>
-    );
+	return (
+		<ModalContainer title="Contributors">
+			{mainContributors.map((mainContributor, index) => {
+				return (
+					<div key={index}>
+						<Styles.contributorImage
+							src={mainContributor.image}
+							alt={mainContributor.name}
+						/>
+						<Styles.personDatas>
+							<Styles.personData>{mainContributor.name}</Styles.personData>
+							<Styles.personData>
+								<b>{mainContributor.function}</b>
+							</Styles.personData>
+							<Styles.personData>
+								{mainContributor.institution}
+							</Styles.personData>
+						</Styles.personDatas>
+					</div>
+				);
+			})}
+
+			<br></br>
+			<h2>Developers:</h2>
+			<br></br>
+			{otherContributors.map((otherContributor, index) => {
```

Comment:
It's better you segregate this `otherContributors` in a array with `developers` and `researchers`. Why? Imagine new developers join in our team, we will need to update the index value all the time. So it's better have another map running into a array then this littles bugs for the future

Reply:
Got it! I'll take a look into it.